shared/src/state-helpers/create-vehicle-parameters.ts

```diff
@@ -31,14 +33,16 @@ export function createVehicleParameters(
         Material.generateMaterial(
             materialTemplates[currentMaterial],
             vehicleId,
-            vehicleTemplate.name
+            vehicleTemplate.name,
+            MapPosition.create(vehiclePosition)
         )
     );
     const personnel = vehicleTemplate.personnel.map((currentPersonnel) =>
         Personnel.generatePersonnel(
             personnelTemplates[currentPersonnel],
             vehicleId,
-            vehicleTemplate.name
+            vehicleTemplate.name,
+            MapPosition.create(vehiclePosition)
```

Comment:
I believe the material and personnel should, by default be loaded into the vehicle that is generated below. This would also preserve the previous behavior.

Reply:
Done